src/BuildScriptGenerator/Node/NodePlatform.cs

```diff
@@ -28,6 +28,10 @@ namespace Microsoft.Oryx.BuildScriptGenerator.Node
         PruneDevDependenciesPropertyKey,
         "When using different intermediate and output folders, only the prod dependencies are copied to the output. " +
         "Options are 'true', blank (same meaning as 'true'), and 'false'. Default is false.")]
+    [BuildProperty(
+        RequireBuildPropertyKey,
+        "Requires either 'npm run build' or 'yarn run build' or custom run build command  to be run. Default is false. " +
+        "If value is not provided, it is assumed to be 'true'.")]
```

Comment:
nit: im bit confused with this sentence:  "If value is not provided, it is assumed to be 'true'.", shouldn't it be the other way round?

Reply:
This meant the scenario: `oryx build /app -p require_build` or `oryx build /app -p require_build=true` are the same. In the first case requiring the build is already part of the name so it is unnecessary to say `true` again. We do the same for prune dev dependencies property.